jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+    def buildlib = release.buildlib
+    def commonlib = release.commonlib
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'COMPONENTS',
+                        description: '(REQUIRED) Brew component name',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "logging-fluentd-container"
+                    ],
+                    [
+                        name: 'ADVISORY',
+                        description: '(REQUIRED) Image release advisory number.',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'RCM_GUEST',
+                        description: 'Details of RCM GUEST',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/redhat/staging-cds/"
+                    ],
+                    [
+                        name: 'MAIL_LIST_SUCCESS',
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'release-engineering@redhat.com',
+                        ].join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+        ]
+    )
+
+    commonlib.checkMock()
+    buildlib.initialize()
+    buildlib.cleanWorkdir(env.WORKSPACE)
+
+    try {
+        sshagent(['openshift-bot']) {
+            advisory = params.ADVISORY ? Integer.parseInt(params.ADVISORY.toString()) : 0
+            if (advisory == 0) {
+                error "Need to provide an ADVISORY ID"
+            }
+            def components = params.COMPONENTS.replaceAll(',', ' ').split(' ')
+            def filename = "tarball-source-output.txt"
+            def outdir = "/mnt/nfs/home/jenkins/container-sources/"
+            def rcm_guest = params.RCM_GUEST.toString()
+            currentBuild.description += "tarball: ${components}"
```

Comment:
Current job description is `nulltarball: [logging-fluentd-container]`. Think `currentBuild.description` is not yet set at this point.